src/open_inwoner/components/templatetags/link_tags.py

```diff
@@ -125,3 +128,59 @@ def addnexturl(href, next_url):
     f = furl(href)
     f.args["next"] = next_url
     return f.url
+
+
+@register.simple_tag(takes_context=True)
+def show_full_dropdown_menu(context) -> bool:
+    """
+    Determines whether the full dropdown menu should be shown.
+
+    This is to avoid showing the full menu in the dropdown when those same items are
+    already displayed in the side menu.
+    """
+    request = context["request"]
+
+    if not request.resolver_match:
+        return True  # Show all items when URL resolution fails
+
+    current_url_name = request.resolver_match.url_name
+    current_namespace = (
+        request.resolver_match.namespaces[0]
+        if request.resolver_match.namespaces
+        else None
+    )
+
+    # Exclude specific namespace:url_name combinations where menu items
+    # are already shown in sidebar or other navigation
+    urls_with_minimal_dropdown_menu = {
+        "pages-root",
+        "general_faq",
+        "collaborate:plan_list",
+        "ssd:uitkeringen",
+        "products:category_list",
+        "inbox:index",
```

Comment:
Is this 'inbox' for our Mijn Berichten chat thing? We do not have a sidenavigation there yet because it already has a sidebar, so it should have the full dropdown menu (which it has, so doesn't seem to do anything here?).

I'm missing Mijn Balie-afspraken UserAppointments(CMSPlugin) here , but not sure yet if that also has a sidenavigation, since i can't get it to work properly locally.

And perhaps we should not wake sleeping dogs but we also have the http://localhost:8000/mijn-profiel/actions/ page that may need a side navigation.

In future we will need sidenaviagtion and limited dropdownmenu for MijnDocumenten and MijnProducten/MijnServices.